packages/lib/har/harUtils.ts

```diff
@@ -64,12 +64,20 @@ export const toHarContentBase64 = (body: Buffer, mimeType?: string): HarFormatCo
   encoding: 'base64',
 });
 
-export const toHarContent = (body: string | Buffer | null, mimeType?: string): HarFormatContent => {
+export const toHarContent = (
+  parseMimeTypesAsJson: string[],
+  body: string | Buffer | null,
+  mimeType?: string,
+): HarFormatContent => {
   if (Buffer.isBuffer(body)) {
     if (isBinary(mimeType ? `file.${extension(mimeType)}` : null, body)) {
       return toHarContentBase64(body, mimeType);
     }
 
+    if (mimeType && parseMimeTypesAsJson.includes(mimeType)) {
```

Comment:
I am wondering if it should be possible to parse the body as JSON even if there is no mime type returned by the server. That could be allowed for example with an empty string in `parseMimeTypesAsJson` (or maybe it is an edge case that is not worth considering). What do you think?

Reply:
I think that it's a realistic edge case, I will add that in and include it in the docs